installation/scripts/minikube.sh

```diff
@@ -6,7 +6,9 @@ CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 RESOURCES_DIR="${CURRENT_DIR}/../resources"
 
 MINIKUBE_DOMAIN=""
-MINIKUBE_VERSION=0.33.0
+# Supported Minikube Versions: MINIKUBE_VERSION_MIN (inclusive) up to MINIKUBE_VERSION_MAX (exclusive)
+MINIKUBE_VERSION_MIN=0.33.0
+MINIKUBE_VERSION_MAX=0.34.0
 KUBERNETES_VERSION=1.11.5
```

Comment:
Please do not introduce max version of k8s, we do not have an infra to test that. Instead, you should print a warning if you're using a different version than the one in our resources. Here you can limit the check to minor and skip patch.

Reply:
done